src/main/java/org/chsrobotics/lib/drive/differential/DifferentialDrivetrainInput.java

```diff
@@ -86,10 +85,8 @@ public String toString() {
     public boolean equals(Object o) {
         if (this == o) return true;
         if (o == null || getClass() != o.getClass()) return false;
-        double epsilon = 0.000001;
         DifferentialDrivetrainInput that = (DifferentialDrivetrainInput) o;
-        return Precision.equals(that.left, left, epsilon)
```

Comment:
.equals() is something that's meant for unit tests, really, not actual use. The tests have been consistent so far so I don't think this will be an issue.

Reply:
I disagree that it’s not for actual use and since we’ve added apache math why not add back